.github/workflows/pull_request.yaml

```diff
@@ -45,9 +45,8 @@ jobs:
     - name: Run tests
       env:
         GITHUB_TOKEN: ${{ secrets.CLOUDBUILD_GITHUB_TOKEN }}
-      run: |
-        go install -v github.com/onsi/ginkgo/ginkgo@v1.16.5 && export PATH=$PATH:$(go env GOPATH)/bin/
-        ginkgo -r -failFast -randomizeSuites -randomizeAllSpecs -skipPackage=./installutils/kubeinstall,./debugutils/test
+        TEST_PKG: ./... # Run all tests
+      run: make test
     - uses: testspace-com/setup-testspace@v1
       with:
```

Comment:
what goes on, here? Specifically: we now have some newly modified code coverage+junit report stuff. I've never actually seen this testspace server/data

I'm asking, as I think it might be more useful to store these as github actions artifacts

Reply:
We used to execute a ginkgo command with several arguments in a github action. What I have done is migrate that command to a make target to mirror our other repos. This way, devs can execute the same command that is run in CI (`make test`). I updated the flags we provide to use the new v2 syntax but did not change it apart from that.